modules/serverless-logic-job-service-custom-execution.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-job-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-job-service-custom-execution_{context}"]
+= Running custom execution
+
+To run the Job service, select one of the following images based on the persistence mechanism you are using PostgreSQL or Ephemeral:
+
+* `registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8`
+* `registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8`
+
+[NOTE]
+====
+The common configurations and the eventing API configurations are the same for both images.
+====
+
+.Prerequisites
+
+* You have {ServerlessLogicOperatorName} installed on your cluster.
+* You have access to a {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* You have installed the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Identify the image to use depending on the persistence mechanism, and see the required configuration parameters specific for that image.
+. Identify if you need the Eventing API, and check the required configuration parameters for it.
+. Ensure that the project containing your workflows is set up with the correct Job service Quarkus extension.
```

Comment:
Could you link here with the reference to these Quarkus extensions or expand in more detail?

Reply:
At this point, we are not going to elaborate the full quarkus project maven programmed work mode.
In fact, open question above is if we still include all this custom JS execution documentation fragments, or just say that the JS is installed by the operator, a and redirect users to this upcoming section.

Reply:
Let me know if I can remove this section.

Reply:
yes we can